features/full_tests/native_event_tracking.feature

```diff
@@ -5,7 +5,7 @@ Feature: Synchronizing app/device metadata in the native layer
         And I wait to receive an error
         Then the error payload contains a completed handled native report
         And the event "app.inForeground" is true
-        And the event "app.duration" is greater than 0
+        And the event "app.duration" is not null
```

Comment:
Rather than just asserting _something_ is here, I'd add a new assertion to check that the stringified value is sensible. i.e.
```cucumber
And the event "app.duration" is a stringified number greater than 0
```

and check that the value is something like `/^[1-9][0-9]+$/` and `value.to_i > 0`